cli/src/commands/xlr/compile.ts

```diff
@@ -141,6 +139,18 @@ export default class XLRCompile extends BaseCommand {
                 return;
               }
 
+              // Get registration name of plugin
+              node.members.forEach((member) => {
```

Comment:
would this be better with `node.members.find` rather than a `forEach` if you just need the name from whichever member has it?

Reply:
Good point! Let me simplify that.